config/config.exs

```diff
@@ -25,8 +25,7 @@ config :mobile_app_backend, MBTAV3API.RepositoryCache,
 # Configures the endpoint
 config :mobile_app_backend, MobileAppBackendWeb.Endpoint,
   url: [host: "localhost"],
-  http: [compress: true],
```

Comment:
note: compress is[ true by default ](https://hexdocs.pm/bandit/Bandit.html#t:http_options/0)